_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<h1>Menu</h1>
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+				</li>
+			</ul>
+					
+		</div>
+	</div>
+</body>
+```
+
+
+#### Passed Example 4
+
+This document is using the word "square" but in this case it is no instruction.
+
+```html
+<body>
+	<p>In mathemathics a square is the result of multiplying a number by itself.</p>
+</body>
+```
+
+#### Passed Example 5
+
+The following text is tilted.
+
+```html
+<head>
+	<style> 
+		div.tilt {
+			height: 750px;
+			width: 150px;
+			-ms-transform: rotate(20deg); /* IE 9 */
+			-webkit-transform: rotate(20deg); /* Safari 3-8 */
+			transform: rotate(20deg);
+			}
+	</style>
+</head>
+<body>
+	<div class="tilt">This text is tilted.</div>
+</body>
+```
+
+#### Passed Example 6
+
+The button is indicated by the word "round". But the word is also included in the text of the element.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+		border-radius: 50%;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the round button, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Round button</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 7
+
+The image is indicated by the word narrow but the alt text of the image also includes the word narrow.
+
+```html
+<body>
+	<p>The wide image is awesome. But the narrow image isn't.</p>
+	<img scr="/test-assets/images/awesome_wide.jfif" alt="Wide photo of an awesome landscape.">
+	<img scr="/test-assets/images/Non_awesome_narrow.jpg" alt="Narrow photo of a dull landscape.">
+	</div>
+</body>
+```
+
+#### Passed Example 8
+
+This document is using the word "triangle" but in this case the triangle menu is on a different page and has a heading "triangle menu".
+
+```html
+<body>
+	<p>On the <a href="/test-assets/SC1.3.3-triangle-menu-with-heading.html">information page</a> you can find more examples within the triangle menu</p>
+</body>
+```
+
+#### Passed Example 9
+
+This document is using the word "star" but in this case the star is in an iframe and has a heading "examples".
+
+```html
+<body>
+<p>More examples can be found when you look underneath the star or underneath the heading examples</p>
```

Comment:
Is "underneath the heading" a visual reference?

Also, "examples" should probably be "Examples" (as in the actual code), and I would probably put it in quotation marks, or in italic, to show that it is the actual name.